Add download directories (#39)
Nesper94/Metabiome: docs/source/tutorial.rst

contaminants are sequences coming from researchers or people manipulating the
samples and sequences from the Phi-X174 phage used as control in the
sequencing machines, so we will remove reads coming from these sources using
`Bowtie2 <http://bowtie-bio.sourceforge.net/bowtie2/index.shtml>`_.

Before running :code:`bowtie2` command let's download through the next links the
`subsampled Human reference Genome <https://drive.google.com/file/d/1f49lWDaX63FefH150PZ_p9FUa5UwE5zk/view?usp=sharing>`_
and the `Phi-X174 genome <https://drive.google.com/file/d/1uRdEzysZCySSkBqp-uEn-Cx5MbsQ5F8n/view?usp=sharing>`_,

> which we will use to decontaminate the filtered reads.Also, place the subsampled

Reviewer: Add space after period:
```suggestion
which we will use to decontaminate the filtered reads. Also, place the subsampled
```

However, due to tutorial purposes only, you will have to download our custom
database located here: `metaphlan3_custom_db <https://drive.google.com/drive/folders/1xNzSYTjSYlfycDsSC6_QM47y9Yid9Oe5?usp=sharing>`_.
Be aware that this database is compressed and after downloading it, you must

> extract the folder :file:`metaphlan_custom_db.tar.gz` :

Reviewer: Remove space after link:
```suggestion
extract the folder :file:`metaphlan_custom_db.tar.gz`:
```

**************

The following step is to generate bins from the previous draft genomes or

> contigs. To do so, we will use the contigs from MetaSPAdes's output through three

Reviewer: I think it is good to keep the `(wether from MetaSPAdes or MEGAHIT)` part, but stating that in this tutorial the results from **MEGAHIT** will be used (If we say MetaSPAdes then we would have to change the following parts of the tutorial). It would be also necessary to change **wether** for **whether** or use **either** instead.
```suggestion
contigs (either from MetaSPAdes or MEGAHIT). In this tutorial, we will
use the contigs from MEGAHIT's output through three
```
